extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java

```diff
@@ -145,14 +181,31 @@ private KafkaBroker startKafka(KafkaDevServiceCfg config) {
         }
 
         if (!isDockerWorking.getAsBoolean()) {
-            log.warn("Docker isn't working, please configure the Kafka bootstrap servers property (kafka.bootstrap.servers).");
+            log.warn(
+                    "Docker isn't working, please configure the Kafka bootstrap servers property (kafka.bootstrap.servers).");
             return null;
         }
 
+        if (config.shared && launchMode.getLaunchMode() == LaunchMode.DEVELOPMENT) {
+            // Detect if there is a broker already started using container labels.
+            Container container = lookup(config.serviceName);
+            if (container != null) {
+                log.infof("Dev Services for Kafka container found: %s (%s)", container.getId(),
```

Comment:
Move this log message into the `if` below? Because it seems that at this point, it is _not_ clear that the shared broker will be used. Nice side effect is that there's gonna be only one log message, which is more than enough :-)

Reply:
Fixed.